index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/4a30d59e86.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <div id="body-wrapper">
+      <nav id="nav-bar">
+        <div id="logo">My Logo</div>
+        <div id="menu">
+          <i class="fa-solid fa-bars"></i>
+        </div>
+      </nav>
+      <main>
+        <section id="headline">
+          <img
+            src="./images/first-section/Ellipse%203.svg"
+            alt="bg-img"
+            class="ell-3"
+          />
+          <img
+            src="./images/first-section/Ellipse%204.svg"
+            alt="bg-img"
+            class="ell-4"
+          />
+          <img
+            src="./images/first-section/Ellipse%205.svg"
+            alt="bg-img"
+            class="ell-5"
+          />
+          <img
+            src="./images/first-section/Ellipse%208.svg"
+            alt="bg-img"
+            class="ell-8"
+          />
+          <img
+            src="./images/first-section/Ellipse%2011.svg"
+            alt="bg-img"
+            class="ell-11"
+          />
+          <img
+            src="./images/first-section/Ellipse%2012.svg"
+            alt="bg-img"
+            class="ell-12"
+          />
+          <img
+            src="./images/first-section/Ellipse%2013.svg"
+            alt="bg-img"
+            class="ell-13"
+          />
+          <img
+            src="./images/first-section/Ellipse%2014.svg"
+            alt="bg-img"
+            class="ell-14"
+          />
+          <img
+            src="./images/first-section/Ellipse%2015.svg"
+            alt="bg-img"
+            class="ell-15"
+          />
+          <img
+            src="./images/first-section/Rectangle%204.svg"
+            alt="bg-img"
+            class="rec-4"
+          />
+          <img
+            src="./images/first-section/Rectangle%205.svg"
+            alt="bg-img"
+            class="rec-5"
+          />
+          <img
+            src="./images/first-section/Rectangle%206.svg"
+            alt="bg-img"
+            class="rec-6"
+          />
+          <img
+            src="./images/first-section/Rectangle%207.svg"
+            alt="bg-img"
+            class="rec-7"
+          />
+
+          <h2 id="headline-text">I'm Kanza. Glad to see you!</h2>
+          <p id="supporting-text">
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hestiate to
+            contact me.
+          </p>
+
+          <p id="connect">LET'S CONNECT</p>
```

Comment:
- [ ] Please make sure to follow the Semantic HTML principles and make the connect button as a `<button>` tag